src/lib.rs

```diff
@@ -371,7 +371,7 @@ mod tests {
     #[cfg(feature = "gdk-pixbuf")]
     fn decode_blurhash_pixbuf() {
         use std::convert::TryInto;
-        let img = gdk_pixbuf::Pixbuf::from_file("data/octocat.png").unwrap();
+        let img = gdk_pixbuf::Pixbuf::from_file("data/wikipedia_logo.svg").unwrap();
```

Comment:
What's the reason we want to switch to yet another image?

Reply:
The octocat is a square image, so this bug will never happen in that case.